tests/test_context_processors.py

```diff
@@ -13,12 +14,12 @@ def inject_rudolf():
 
         rendered = render_response("context.html")
 
-        expected_data = inspect.cleandoc(
+        expected_data = cleandoc(
             """
             <!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
             <pre>rudolf = The red-nosed reindeer</pre>
             """
-        )
+        ).encode("UTF-8")
```

Comment:
Might be worth leaving a small comment here that the encoding is required due to `render_response` returning bytes

Reply:
Updated with comments!